pkgs/servers/home-assistant/parse-requirements.py

```diff
@@ -189,6 +201,11 @@ def main() -> None:
         f.write("  };\n")
         f.write("}\n")
 
+    if outdated:
+        print("\nOutdated packages:")
+        for package, version in outdated.items():
+            print(f" - {package} {version['nixpkgs']} < {version['hass']}")
```

Comment:
```suggestion
            print(f" - {package}: current: {version['nixpkgs']} < wanted: {version['hass']}")
```

Reply:
I think that makes it unnecessarily noisy:

```
Outdated packages:
 - bt_proximity: current: 0.0.20180217 < wanted: 0.2
 - brother: current: 0.1.18 < wanted: 0.1.20
 - plexapi: current: 4.1.2 < wanted: 4.2.0
 - plexauth: current: 0.0.5 < wanted: 0.0.6
 - py-cpuinfo: current: 5.0.0 < wanted: 7.0.0
 - fixerio: current: 0.1.1 < wanted: 1.0.0a0
 - linode-api: current: 4.1.8b1 < wanted: 4.1.9b1
 - libsoundtouch: current: 0.4.0 < wanted: 0.8
 - python-telegram-bot: current: 13.0 < wanted: 13.1
 - holidays: current: 0.10.3 < wanted: 0.10.4
```

Maybe a table would be better.

Reply:
```
             Outdated dependencies              
┏━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━━━┓
┃ Package             ┃ Current      ┃ Wanted  ┃
┡━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━╇━━━━━━━━━┩
│ bt_proximity        │ 0.0.20180217 │ 0.2     │
│ brother             │ 0.1.18       │ 0.1.20  │
│ plexapi             │ 4.1.2        │ 4.2.0   │
│ plexauth            │ 0.0.5        │ 0.0.6   │
│ py-cpuinfo          │ 5.0.0        │ 7.0.0   │
│ fixerio             │ 0.1.1        │ 1.0.0a0 │
│ linode-api          │ 4.1.8b1      │ 4.1.9b1 │
│ libsoundtouch       │ 0.4.0        │ 0.8     │
│ python-telegram-bot │ 13.0         │ 13.1    │
│ holidays            │ 0.10.3       │ 0.10.4  │
└─────────────────────┴──────────────┴─────────┘
```

Reply:
fancy.